adapters/quick/src/main/scala/caliban/QuickRequestHandler.scala

```diff
@@ -205,11 +202,24 @@ final private class QuickRequestHandler[-R](interpreter: GraphQLInterpreter[R, A
       .mapConcatChunk(Chunk.fromArray)
   }
 
+  private def encodeTextEventStream(resp: GraphQLResponse[Any])(implicit trace: Trace) = {
+    val stream = (resp.data match {
+      case ObjectValue((fieldName, StreamValue(stream)) :: Nil) =>
+        stream.either.map {
+          case Right(r)  => GraphQLResponse(ObjectValue(List(fieldName -> r)), resp.errors)
```

Comment:
Isn't this saying: for all response bodies return the original errors? Would we want errors to be null here perhaps?

Reply:
Hmm I think you're right. It's probably better to send an initial one with all the errors (if they exist) and then the subsequent ones not containing errors

Reply:
I made the the GraphQLResponse stream handling generic and placed it in HttpUtils in the core module so that both the caliban-quick and tapir adapters use the same one. Also the errors now are reported in the first event only